examples/example_plugin/example_plugin/ui/ModelRetrieveView.jsx

```diff
@@ -0,0 +1,9 @@
+import { useParams } from "react-router-dom";
+
+
+export default function ModelRetrieveView(props){
+    const { pk } = useParams();
+    return (
+            <div>Im a App provided route view with pk: { pk }.</div>
```

Comment:
Please fix spelling/grammar as well as making this a bit more self-documenting as to what this is demonstrating.